.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -109,6 +117,47 @@ body:
         - Not Applicable
     validations:
       required: true
+  
+  - type: dropdown
+    attributes:
+      label: Frontend framework
+      description: Which frontend framework is your app built upon?
+      multiple: false
+      options:
+        - "Angular"
+        - "Gatsby"
+        - "Hugo"
+        - "JavaScript"
+        - "Jekyll"
+        - "Nextjs"
+        - "React"
+        - "Remix"
+        - "Svelte"
+        - "Vite"
+        - "Vue"
+    validations:
+      required: true
+      
+
+  - type: input
+    attributes:
+      label: Nextjs version
+      placeholder: Review the next dependency version in your package.json file
+      description: If your app is built using the Nextjs framework, please include the current Nextjs version of your app. If your app is using any other framework, type `N/A`.
```

Comment:
```suggestion
      description: If your app is built using the Next.js framework, please include the current Next.js version of your app. If your app is using any other framework, type `N/A`.
```

Reply:
Should these two items be required fields? If not, the extra explanations can be removed.

Reply:
Yeah I think it makes sense to have these as required fields for Next.js issues on Amplify Hosting. Customers who are using any other framework can select N/A.